examples/morpheusvm/tests/integration/integration_test.go

```diff
@@ -194,7 +194,7 @@ var _ = ginkgo.BeforeSuite(func() {
 	app := &appSender{}
 	for i := range instances {
 		nodeID := ids.GenerateTestNodeID()
-		sk, err := bls.NewSecretKey()
```

Comment:
You can actually leave this as the avalanchego import (this is for Warp, which uses that lib)